desc/compute/_stability.py

```diff
@@ -398,11 +398,27 @@ def _ideal_ballooning_gamma2(params, transforms, profiles, data, **kwargs):
 
     A_redo = B_inv @ A @ jnp.transpose(B_inv, axes=(0, 1, 3, 2))
 
-    w, _ = jnp.linalg.eigh(A_redo)
-    # max over "zeta" axis, still a function of rho, alpha, zeta0
-    gamma = jnp.real(jnp.max(w, axis=(2,)))
+    # TODO: Issue #1750
+    # Try jax.scipy.eigh_tridiagonal for increased performance
```

Comment:
is A tridiagonal? I thought it was dense?

Also FYI I've used eigh_tridiagonal for some other stuff, and on CPU it can be faster but its purely sequential so its pretty bad on GPU. also back when I tried it, it could only give eigenvalues, not eigenvectors.